CHANGELOG.md

```diff
@@ -61,6 +61,7 @@
 - Minor: Migrated /ban to Helix API. (#4049)
 - Minor: Migrated /timeout to Helix API. (#4049)
 - Minor: Migrated /w to Helix API. Chat command will continue to be used until February 11th 2023. (#4052)
+- Minor: Migrated /vips to Helix API. (#4053)
```

Comment:
```suggestion
- Minor: Migrated /vips to Helix API. Chat command will continue to be used until February 11th 2023. (#4053)
```

Reply:
Applied in 48873214508426063cb93a88e0c1fb59f365c75a